lib/expr.c

```diff
@@ -1182,13 +1181,11 @@ grn_expr_append_obj(grn_ctx *ctx, grn_obj *expr, grn_obj *obj, grn_operator op,
     case GRN_OP_AND :
     case GRN_OP_OR :
     case GRN_OP_AND_NOT :
+    case GRN_OP_ADJUST :
       PUSH_CODE(e, op, obj, nargs, code);
       if (nargs != 2) {
         GRN_LOG(ctx, GRN_LOG_WARNING, "nargs(%d) != 2 in relative op", nargs);
       }
-      if (obj) {
-        GRN_LOG(ctx, GRN_LOG_WARNING, "obj assigned to relative op");
```

Comment:
Why we can remove this?
Is `if (obj)` never true?

Reply:
We use the `obj` to pass weight factor.

Reply:
I see.